keras_nlp/src/utils/preset_utils.py

```diff
@@ -134,14 +140,16 @@ def get_file(preset, path):
             else:
                 raise ValueError(message)
 
-    elif preset.startswith(GS_PREFIX):
+    elif any(preset.lower().startswith(scheme + "://") for scheme in tf.io.gfile.get_registered_schemes()):
         url = os.path.join(preset, path)
-        url = url.replace(GS_PREFIX, "https://storage.googleapis.com/")
-        subdir = preset.replace(GS_PREFIX, "gs_")
+        subdir = preset
+        for scheme in tf.io.gfile.get_registered_schemes():
```

Comment:
can't this whole block just be something like

`subdir = subdir.replace("://", _").replace("/", "_").replace("-", "_")`